demo/utils/CopyCodeButton.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <n-button
+    class="edit-button"
+    text
+    :size="size"
+    @click="handleClick"
+    :depth="depth"
+  >
+    <template #icon>
+      <n-icon>
+        <copy-icon />
+      </n-icon>
+    </template>
+  </n-button>
+</template>
+
+<script>
+import CopyIcon from '@vicons/fluent/Copy24Regular.js'
```

Comment:
Use Copy16 instead. It's better for small icon.

Reply:
![image](https://user-images.githubusercontent.com/26664586/121197722-1f48f980-c8a4-11eb-8a27-3d68a8a76c70.png)

16 looks smaller than the others